### DISCUSSION TITLE
Ayuda para su practica de css

### DISCUSSION THREAD

**Discussion by Evelo00** [Launch-X-Latam/MisionFrontEnd#366] [Ideas]
Hola a todos!
Me tome la molestia de recrear en Figma la practica de CSS = Vacunacion.
Aqui les dejo el link que puede ayudarles para facilitar su trabajo, pueden tomar de referencia los espaciados del figma, me base en la imagen que fue publicada en la seccion de practicas
PD: Subo una copia por si llega a dañarse, de igual forma procuren realizar un clon y de ahi seguir su trabajo.

LINK 
https://www.figma.com/file/8n6ntXBaRjtPF5Ho8NpV64/Covid-LauncherX-(Copy)

**Comment by themelonkitty** [comment]
Muchas gracias :)

**Comment by jlnunezobando** [comment]
Muchas gracias
